src/app/core/store/reducers/defendants.reducer.ts

```diff
@@ -8,6 +8,7 @@ export interface DefendantsState {
   loading: { [id: number]: boolean };
   deleted: { [id: number]: boolean };
   deleting: { [id: number]: boolean };
+  assetsToDefendantsCount: { [id: number]: number };
```

Comment:
We should not save in the store a derived state that can also be fetched from the state as a selector.